### DISCUSSION TITLE
Use without Expo

### DISCUSSION THREAD

**Discussion by 0xRowdy** [founded-labs/react-native-reusables#221] [Q&A]
Before I jump in, I was wondering if this can be used without Expo. It didn't look like it was supported in the docs, but I figured I'd ask.

**Comment by RWOverdijk** [comment]
I think you can just replace `expo install` with `npm/yarn/whatever install`. From what I can see, most of it is react native, not expo. 

Obviously the web specific stuff won't work without react native web. And you'll have to follow the manual installation. But if you're going react native, expo really is the easiest way to go. You can still do all your native stuff and just use expo prebuild to have it generate the native project for you.

**Comment by mrzachnugent** [comment]
If you can use NativeWind, you can use react-native-reusables. The main difference is with the installation command and the `_layout.tsx` file (the root component). You will have to change the way you do those but you can still do it without expo.

**Reply by marouanechoukri31** [reply]
I’m using a Vite + React application, and I want to set it up without Expo. Initially, I encountered an issue with the error ‘The JSX syntax extension is not currently enabled", I resolved it with this StackOverflow post: https://stackoverflow.com/questions/74620427/how-to-configure-vite-to-allow-jsx-syntax-in-js-files.

Now, I’m facing another issue: the styles aren’t being applied to the components. My Tailwind configuration is working—I tested it with \<div className="bg-red-500 text-white p-4">Test!\</div>, and it displays correctly.

What could be causing the styles not to display? Is it mandatory to implement a \<ThemeProvider /> just to see the styles?

Here’s my repo: https://github.com/marouanechoukri31/react-rnr-tailwind.

 I appreciate your help!

**Reply by mrzachnugent** [reply]
Hey @marouanechoukri31, I will look into it later today or this weekend. Did you try using version `1.0.5`? I haven't tried using them from a vite + react application yet.

**Reply by marouanechoukri31** [reply]
I just forgot to add "jsxImportSource": "nativewind" in my tsconfig, know it's working. thank's for your help.
